api/scripts/protobuf-bindings-generator.sh

```diff
@@ -185,6 +185,8 @@ generate_golang_bindings() {
                 "${go_grpc_out_flag}" \
                 "${go_grpc_module_flag}" \
                 "${go_grpc_unimplemented_servers_flag}" \
+                "${connect_go_grpc_flag}" \
+                "${connect_go_module_flag}" \
```

Comment:
So, do we need to generate both the connect_go bindings and the "normal" bindings? I haven't looked at the generated files. Are connect_go bindings a layer on top of the normal bindings?

Reply:
Yes; we would need gRPC and connect bindings for Go. Because our clients still going to be using go-gRPC. Once we switch them to use connect, we can remove go-grpc bindings. Conenct just creates a new bindings that is backward compatible with go-grpc.